### ISSUE TITLE
Remove deprecated twisted.protocols.loopback.loopback

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4547]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4547|
|Type|task|
|Created|2010-07-05 23:52:47Z|
|Branch|https://github.com/twisted/twisted/tree/remove-loopback-4547|

```loopback``` in [twisted/protocols/loopback.py](twisted/protocols/loopback.py) has been deprecated since Twisted 2.5, let's remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__4547 4547
type__task task
reporter__thijs thijs
priority__lowest lowest
milestone__ 
branch__branches_remove_loopback_4547 branches/remove-loopback-4547
branch_author__cyli cyli
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1278373967000000 1278373967000000
changetime__1295215104000000 1295215104000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_2"></a>|@cyli commented|
|-|-|

(In [30431]) Branching to 'remove-loopback-4547'

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli commented|
|-|-|

(In [30432]) Removed twisted.protocols.loopback.loopback and its accompanying test.

refs [#4547](https://github.com/twisted/twisted/issues/4547)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli removed owner|
|-|-|

Build results:  http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-loopback-4547

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @cyli|
|-|-|

Looks good, except the news file is in the wrong place - should be twisted/topfiles/ instead of twisted/news/topfiles

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_6"></a>|@cyli commented|
|-|-|

Will move it in the merge

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_7"></a>|@cyli set status to `closed`|
|-|-|

(In [30471]) Merge branch remove-loopback-4547: remove t.protocols.loopback.loopback

Author: cyli
Reviewer: exarkun
Fixes [#4547](https://github.com/twisted/twisted/issues/4547)

Remove deprecated (since 2.5) twisted.protocols.loopback.loopback

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
